spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -685,6 +685,32 @@ def setModelData(self, editor, model, index):
         self.set_value(index, value)
 
 
+class BaseHeaderView(QHeaderView):
+    """
+    A header view for the BaseTableView that emits a signal when the width of
+    one of its section is resized by the user.

```

Comment:
Please change `section` for `sections` here (it's the grammatically correct expression in the context of this sentence).